frontend/app/components/button-link.tsx

```diff
@@ -32,7 +32,7 @@ type ButtonLinkStyleProps = {
   variant?: keyof typeof variants;
 };
 
-type ButtonLinkProps = ComponentProps<typeof AppLink> &
+export type ButtonLinkProps = ComponentProps<typeof AppLink> &
```

Comment:
Nitpick: this isn't *bad*, but you could also just do `ComponentProps<ButtonLink>` wherever you need to use this (like we do elsewhere).. just sayin.

Will resolve this comment because it truly is a nitpick.